languagetool-language-modules/sr/src/main/resources/META-INF/org/languagetool/language-module.properties

```diff
@@ -0,0 +1 @@
+languageClasses=org.languagetool.language.Serbian,org.languagetool.language.SerbiaSerbian,org.languagetool.language.BosniaSerbian,org.languagetool.language.CroatiaSerbian,org.languagetool.language.MontenegroSerbian,
```

Comment:
I guess the comma at the end should be removed.

Reply:
Fixed, pending commit.